templates/ipsec.conf

```diff
@@ -21,7 +21,8 @@ conn vpnpsk
   type=transport
   ike=3des-sha1,aes-sha1
   ikev2=no
-  phase2alg=3des-sha1,aes-sha1
+  phase2alg=3des-sha1,aes-sha1,aes256-sha2
+  sha2-truncbug=no
```

Comment:
Once this change is made, does this mean the VPN is no longer compatible with OSX Catalina?

Reply:
I'd have to contact the customer again and get more information about the old device and if he can connect again using it.

Reply:
This has now been tested on Catalina and Big Sur.